util.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
```

Comment:
I think we need a `set -e` at the top here.

Currently, I am seeing failures when trying to move the tar to the remote but for it then still tries to untar anyways.  The script / command should fail if one part of the command fails.